### PR TITLE
Integrate harvester UI

### LINKED ISSUE #4792: Port Harvester UI elements to Flask App
## User Story

In order to streamline our codebase, datagovteam wants incorporate the templates created in `datagov-harvest-ui` into the `datagov-harvest-admin` app.

These elements will not require a login and will be read-only.

Adding these features to our Flask app allows us to shrink our application footprint (by removing the datagov-harvest-ui app from the stack), and allows us to leverage the already-created Flask backend.

## Acceptance Criteria

[ACs should be clearly demoable/verifiable whenever possible. Try specifying them using [BDD](https://en.wikipedia.org/wiki/Behavior-driven_development#Behavioral_specifications).]

- [ ] GIVEN I have view logic that has been built in datagov-harvest-ui
  THEN I want to port that to the datagov-harvest-admin app

## Background

To install packages only available via NPM (ex. USWDS), follow this guide here: https://stackoverflow.com/questions/24514936/how-can-i-serve-npm-packages-using-flask

[Any helpful contextual notes or links to artifacts/evidence, if needed]

## Security Considerations ([required](https://nvd.nist.gov/800-53/Rev4/control/CM-4))

[comment]: # "Our SSP says 'The Data.gov team ensures security implications are considered as part of the agile requirements refinement process by including a section in the issue template used as a basis for new work.' so please don't remove this section without care."
[Any security concerns that might be implicated in the change. "None" is OK, just be explicit here!]

## Sketch

- [ ] port all template work done in datagov-harvest-ui into datagov-harvest-admin
- [ ] hook into pre-existing API endpoints to fetch required data
- [ ] confirm none of the view templates requires a login

### PR DESCRIPTION
**Pull request by btylerburton**
# Pull Request

Resolves https://github.com/GSA/data.gov/issues/4792

## About

Integrates Harvest UI elements from https://github.com/GSA/datagov-harvest-ui

## PR TASKS

- [X] The actual code changes.
- [X] Tests written and passed.
- [ ] Any changes to docs?

### CODE REVIEW

**Comment by btylerburton** [comment]
Comments out the session tests until I can isolate the GH Actions issue

File: app/routes.py [hunk 1]
```diff
@@ -351,54 +351,100 @@ def add_harvest_source():
...
+                "backgroundColor": "black",
+            },
+            {
+                "label": "Errored",
+                "data": [job.records_errored for job in jobs],
+                "borderColor": "red",
+                "backgroundColor": "red",
+            },
+            {
+                "label": "Igrored",
```
Comment (rshewitt) [inline app/routes.py]: typo. not trying to nitpick just wanna make sure your stuff gets looked at!

Comment (btylerburton) [reply app/routes.py]: good catch!

**Comment by btylerburton** [comment]
Session tests fixed. 🚀

File: .github/workflows/commit.yml [hunk 2]
```diff
@@ -22,7 +22,6 @@ jobs:
         CF_SERVICE_AUTH: ${{secrets.CF_SERVICE_AUTH}}
         HARVEST_RUNNER_APP_GUID: ${{secrets.HARVEST_RUNNER_APP_GUID}}
         FLASK_APP_SECRET_KEY: ${{secrets.FLASK_APP_SECRET_KEY}}
-        OPENID_PRIVATE_KEY: ${{secrets.OPENID_PRIVATE_KEY}}
```
Comment (btylerburton) [inline .github/workflows/commit.yml:25]: Removing this proactively as we don't need it for testing and it's not stored in GH as a secret.

**Comment by btylerburton** [comment]
Just remembered that I need to add a new step to pipeline to build the static assets. This shouldn't hold up this ticket but it will hold up resolution of the issue.

**Comment by btylerburton** [comment]
> Just remembered that I need to add a new step to pipeline to build the static assets. This shouldn't hold up this ticket but it will hold up resolution of the issue.

✅ Added new static assets build step

File: app/routes.py [hunk 3]
```diff
@@ -245,56 +238,72 @@ def add_organization():
...
-            "edit_data.html",
-            form=form,
-            action="Edit",
-            data_type="Organization",
-            button="Update",
-        )
-    else:
-        org = db.get_all_organizations()
-    return org
+def edit_organization(org_id):
```
Comment (rshewitt) [inline app/routes.py:281]: +1 for removing default

File: app/routes.py [hunk 4]
```diff
@@ -351,54 +361,105 @@ def add_harvest_source():
...
+    return render_template("view_source_list.html", data=data)
 
 
 # Edit Source
-@mod.route("/harvest_source/edit/<source_id>", methods=["GET", "POST"])
+@mod.route("/harvest_source/config/edit/<source_id>", methods=["GET", "POST"])
 @login_required
-def edit_harvest_source(source_id: str = None):
+def edit_harvest_source(source_id: str):
     if source_id:
```
Comment (rshewitt) [inline app/routes.py:425]: do we still need this condition if `source_id` is required?

File: app/routes.py [hunk 5]
```diff
@@ -351,54 +361,105 @@ def add_harvest_source():
...
-        return render_template(
-            "edit_data.html",
-            form=form,
-            action="Edit",
-            data_type="Harvest Source",
-            button="Update",
-            source_id=source_id,
-        )
+        source = db.get_harvest_source(source_id)
+        organizations = db.get_all_organizations()
```
Comment (rshewitt) [inline app/routes.py:427]: we have 133. that number probably isn't going to increase enough to make this call expensive. just a comment.

File: app/routes.py (hunk 4 above)
Comment (btylerburton) [reply app/routes.py:425]: good point. probably not. i'll make the update.

Comment (btylerburton) [reply app/routes.py:425]: I'm going to defer this until the route cleanup ticket, if that's ok with you. There's some similar logic that exists in harvest_jobs and harvest_records that is inconsistent which should also be cleaned up.

**Comment by rshewitt** [comment]
LGTM!

**Review by rshewitt — APPROVED** [review]

**Pull request merged**